tests/integration/api/conftest.py

```diff
@@ -77,12 +86,13 @@ def check_permission_mock():
 
 
 @pytest.fixture
-def default_de() -> models.DecisionEnvironment:
+def default_de(default_organization) -> models.DecisionEnvironment:
     """Return a default DE."""
     return models.DecisionEnvironment.objects.create(
         name="default_de",
         image_url="quay.io/ansible/ansible-rulebook:latest",
         description="Default DE",
+        organization=default_organization,
```

Comment:
This is fine, but for the record, you should get the same behavior if you use the `default_organization` as a fixture but _don't_ pass it as a kwarg here.

Reply:
Ah makes sense. I'll remove the args then.